nextjs/src/app/(app)/data-sources/create/configure/[externalDataSourceId]/page.tsx

```diff
@@ -16,7 +16,9 @@ import {
 import { toast } from "sonner";
 import { UpdateMappingForm } from "@/components/UpdateMappingForm";
 import { LoadingIcon } from "@/components/ui/loadingIcon";
-import {UDPATE_EXTERNAL_DATA_SOURCE} from '@/graphql/mutations';
+import { UDPATE_EXTERNAL_DATA_SOURCE } from '@/graphql/mutations';
+import { triggerCustomEvent } from "../../../../../../app/utils/posthogutils"; // Adjust the path as needed
```

Comment:
Let's normalise this import path to @/utils/posthogutils or whatever it's meant to be.